GitLab/commands.yaml

```diff
@@ -0,0 +1,26 @@
+sourceBasePath: ''
+commands:
+  stats:
+    sourcePath: /stats.js
```

Comment:
We are normalizing the location of all functions following Nimbella project layout.
So in this case, functions should be located in a folder called `packages/gitlab` and the name of the file for this function will be `gitlab_stat.js` (And so on below).

Take a look at this PR for an reference https://github.com/nimbella/command-sets/pull/54/files.